### PR TITLE
onboarding exercise

### PR DESCRIPTION
**Pull request by onuruci**

### CODE REVIEW

**Review by derrix060 — COMMENTED** [review]
Not a juno developer, but it would be nice to have more details in the commit message and some documentations for the functions :)

**Comment by onuruci** [comment]
Just completed onboarding exercises this is not to be merged

File: rpc/class_test.go [hunk 1]
```diff
@@ -218,3 +218,23 @@ func TestClassHashAt(t *testing.T) {
...
+
+func TestJunoGetNodesFromRoot(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
+	t.Cleanup(mockCtrl.Finish)
+
+	mockReader := mocks.NewMockReader(mockCtrl)
+	log := utils.NewNopZapLogger()
+	handler := rpc.New(mockReader, nil, nil, "", log)
+
+	t.Run("Empty blockchain", func(t *testing.T) {
```
Comment (rianhughes) [inline rpc/class_test.go:230]: It would be nice to have a test with a non-empty blockchain, but otherwise nice work

Comment (onuruci) [reply rpc/class_test.go:231]: Added a new test with non-empty blockchain

**Pull request closed without merging**